pkg/fakerp/config.go

```diff
@@ -75,11 +75,16 @@ func GetPluginTemplate() (*pluginapi.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+	rootDockerConfig, err := readFile(filepath.Join(artifactDir, "root-docker-config.json"))
+	if err != nil {
+		return nil, err
+	}
 	template.Certificates.GenevaLogging.Cert = logCert
 	template.Certificates.GenevaLogging.Key = logKey
 	template.Certificates.GenevaMetrics.Cert = metCert
 	template.Certificates.GenevaMetrics.Key = metKey
 	template.Images.GenevaImagePullSecret = pullSecret
+	template.RootDockerConfig = rootDockerConfig
```

Comment:
I think it is worth moving this under `template.Images.ImagePullSecret` and accordingly `Config.Images.ImagePullSecret`